addons/mail/static/tests/models/user_notification_manager_qunit_tests.js

```diff
@@ -0,0 +1,12 @@
+/** @odoo-module **/
+
+import { patchRecordMethods } from '@mail/model/model_core';
+// ensure that the model definition is loaded before the patch
+import '@mail/models/user_notification_manager';
+
+patchRecordMethods('UserNotificationManager', {
+    _playAudio() {
+        // do nothing during tests.
+    },
```

Comment:
maybe better built-in rather than patched (canPlayAudio false in test or with an extra field?)